### PR TITLE
Manual reference for deep-rules

### PR DESCRIPTION
**Pull request by Benjamin-Lee**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Any more details?**
The current title of reference 4 is "Ten Quick Tips for Deep Learning in Biology. Contribute to Benjamin-Lee/deep-rules development by creating an account on GitHub". While that is the `title` tag of the page, it really isn't the name we want in our references list. I am changing it to conform the example manual reference to the manubot rootstock repo in manual-references.json

### CODE REVIEW

**Comment by Benjamin-Lee** [comment]
Can someone figure out why the build isn't passing? The JSON is valid (according to https://jsonlint.com) so I'm stumped.

**Comment by cgreene** [comment]
@Benjamin-Lee - looks like the build worked now!

**Comment by agitter** [comment]
Yeah, I couldn't find any errors so I restarted the build.  Worked this time.

File: content/manual-references.json [hunk 1]
```diff
@@ -38,5 +38,24 @@
         "literal": "NVIDIA"
     }
   ]
+},
+{
+  "type": "webpage",
+  "standard_citation": "url:https://github.com/Benjamin-Lee/deep-rules",
+  "URL": "https://github.com/Benjamin-Lee/deep-rules",
+  "title": "Benjamin-Lee/deep-rules GitHub repository",
```
Comment (dhimmel) [inline content/manual-references.json]: I might add

```json
"container-title": "GitHub",
```

Comment (agitter) [reply content/manual-references.json]: I agreed with this suggestion and made the change

**Pull request merged**